src/tests/Equinor.ProCoSys.IPO.Query.Tests/GetInvitationsQueries/GetInvitationsForExport/GetInvitationsForExportQueryHandlerTests.cs

```diff
@@ -376,35 +399,6 @@ public async Task HandleGetInvitationsForExportQuery_ShouldFilterOnStatusPlanned
             }
         }
 
-        [TestMethod]
```

Comment:
Why remove this particular test?

Reply:
As far as I understand, the test does a change to an InMemory database which is assumed to be reflected in the results. This is not the case for stored procedure since this is not supported in InMemory database as far as I can see.